models/pardot/model.sql

```diff
@@ -36,13 +36,13 @@ create or replace view pardot_analysis.visitoractivity_types_meta as (
   select 31, 'UserVoice Suggestion' union all
   select 32, 'UserVoice Comment' union all
   select 33, 'UserVoice Ticket' union all
-  select 34, 'Video Watched (≥ 75% watched)'
+  select 34, 'Video Watched (>= 75% watched)'
```

Comment:
this view is clever, but I'm pretty sure it wouldn't be performant if, for example, a huge table was joined against it.  I have another analysis I'm writing that reminds me of this - it needs a "calendar" table full of daily dates from the past and future.  I wrote a generic script to create it, but it really needs to be an actual table, not a view.  This is fine for now, but we should consider handling actual table population somehow.